liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DebugServerMojo.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * (C) Copyright IBM Corporation 2014, 2017.
```

Comment:
Remove the 2014 since this is a new file.  If it is updated in a subsequent year, then it would be updated to 2017, 2018.  You only include the first and last year of change.